docs-source/usersguide/application.rst

```diff
@@ -1022,13 +1022,13 @@ Integrators
 OpenMM offers a choice of several different integration methods.  You select
 which one to use by creating an integrator object of the appropriate type.
 
-Langevin Integrator
--------------------
+BAOAB Langevin Integrator
```

Comment:
Would you mind if I contributed a bit more information to this section?

Reply:
Probably simplest if I merge this, then you can create a new PR with additional text?

Reply:
Sounds good!